projects/GKv3/GEDKeeper3/GKUI/Platform/DropDownToolItem.cs

```diff
@@ -184,18 +185,25 @@ public ContextMenu ContextMenu
 
 #if OS_MACOS
 
-    public class ButtonToolItemHandler : ToolItemHandler<NSToolbarItem, ButtonToolItem>, ButtonToolItem.IHandler
+    public class DropDownToolItemHandler : ToolItemHandler<NSToolbarItem, DropDownToolItem>, DropDownToolItem.IHandler
     {
         ContextMenu contextMenu;
 
+        public new string Text
+        {
+            get { return Button.Title; }
+            set { Button.Title = value; }

```

Comment:
By default text is set to a label underneath the button. We want it to be in the button itself.

Reply:
Shall we change it to only if image is null?